src/test/regress/specs/isolation_ensure_dependency_activate_node.spec

```diff
@@ -0,0 +1,170 @@
+# the test expects to have zero nodes in pg_dist_node at the beginning
+# add single one of the nodes for the purpose of the test
+setup
+{
+    SELECT master_remove_node(nodename, nodeport) FROM pg_dist_node;
+	SELECT nodename, nodeport, isactive FROM master_add_node('localhost', 57637);
+}
+
+# ensure that both nodes exists for the remaining of the isolation tests
+teardown
+{
+    -- schema drops are not cascaded
+    SELECT run_command_on_workers($$DROP SCHEMA IF EXISTS myschema CASCADE;$$);
+    DROP SCHEMA IF EXISTS myschema CASCADE;
+
+    RESET search_path;
+    DROP TABLE IF EXISTS t1 CASCADE;
+    DROP TABLE IF EXISTS t2 CASCADE;
+    DROP TABLE IF EXISTS t3 CASCADE;
+
+	SELECT master_remove_node(nodename, nodeport) FROM pg_dist_node;
```

Comment:
Minor: There are few lines in the spec file that the indentation is off, like this one.